src/main/java/com/stolensugar/web/activity/GetMochiDeckActivity.java

```diff
@@ -0,0 +1,121 @@
+package com.stolensugar.web.activity;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.stolensugar.web.voiceCommands.Mochi.*;
+import com.stolensugar.web.dao.SpokenFormUserDao;
+import com.stolensugar.web.dynamodb.models.SpokenFormUserModel;
+import com.stolensugar.web.model.requests.GetMochiDeckRequest;
+
+import javax.inject.Inject;
+import java.util.ArrayList;
+import java.util.List;
+
+public class GetMochiDeckActivity {
+    private SpokenFormUserDao spokenFormUserDao;
+    @Inject
+    public GetMochiDeckActivity(SpokenFormUserDao spokenFormUserDao) {
+
+        this.spokenFormUserDao = spokenFormUserDao;
+    }
+
+    public String execute (final GetMochiDeckRequest request) throws Exception {
+
+        if(request.getApp() == null || request.getApp() == "talon") {

```

Comment:
instead of this if statement, use [defaultName](https://www.baeldung.com/spring-request-param#a-default-value-for-the-request-parameter) in the controller.